master/internal/command/command.go

```diff
@@ -647,6 +657,22 @@ func (c *command) toV1Job() *jobv1.Job {
 	return &j
 }
 
+func (c *command) persistAndEvictModelDefFromMemory() error {
```

Comment:
would be nice if we persisted and evicted this much earlier.

Reply:
how much earlier? 

Do you mean like in the api calls?

Reply:
yeah, but i dont think it is feasible for this PR